packages/myst-to-react/src/code.tsx

```diff
@@ -88,11 +88,7 @@ export function CodeBlock(props: Props) {
       >
         {value}
       </SyntaxHighlighter>
-      {showCopy && (
-        <div className="absolute hidden top-1 right-1 group-hover:block">
-          <CopyIcon text={value} />
-        </div>
```

Comment:
This was hiding the component, and inaccessbible to tab. Now this is tab-accessibile!